sentry/src/main/java/io/sentry/SentryReplayOptions.java

```diff
@@ -11,6 +11,9 @@ public final class SentryReplayOptions {
 
   public static final String TEXT_VIEW_CLASS_NAME = "android.widget.TextView";
   public static final String IMAGE_VIEW_CLASS_NAME = "android.widget.ImageView";
+  public static final String WEB_VIEW_CLASS_NAME = "android.webkit.WebView";
+  public static final String VIDEO_VIEW_CLASS_NAME = "android.widget.VideoView";
+  public static final String ANDROIDX_MEDIA_VIEW_CLASS_NAME = "androidx.media3.ui.PlayerView";
 
```

Comment:
How about adding the Exoplayer View too? It's `com.google.android.exoplayer2.ui.PlayerView` and `com.google.android.exoplayer2.ui.StyledPlayerView` https://github.com/google/ExoPlayer/blob/release-v2/library/ui/src/main/java/com/google/android/exoplayer2/ui/PlayerView.java

Reply:
Oh I see, media3 is an actual replacement for exoplayer, but I'd still add it.

Reply:
added both 👍